jina/docker/hubio.py

```diff
@@ -208,8 +208,12 @@ def build(self) -> Dict:
                 if self.args.test_uses:
                     try:
                         from jina.flow import Flow
-                        with Flow().add(uses=image.tags[0]):
-                            pass
+                        if self.args.daemon:
```

Comment:
`with Flow().add(uses=image.tags[0], daemon=self.args.daemon):` should suffice?

Reply:
oh, looks like our parser is able to recognize `daemon=False` as not pass anything to the docker, good